sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md

```diff
@@ -136,6 +136,21 @@ There is an [open issue](https://github.com/Azure/azure-sdk-tools/issues/4652) t
 
 For more information on this topic, see the [asset sync reference documentation][asset-sync-reference].
 
+### Through VS Code [GIT Source Control](https://code.visualstudio.com/docs/sourcecontrol/overview) UI
```

Comment:
Use the .md view to view the screenshots
<img width="347" alt="image" src="https://user-images.githubusercontent.com/10452642/233710945-4efbaaae-8677-4df3-a569-dfdd99243711.png">